src/resources/views/crud/columns/relationship_count.blade.php

```diff
@@ -1,14 +1,17 @@
 {{-- relationship_count (works for n-n relationships) --}}
 @php
-    $value = data_get($entry, $column['name'])->count();
-
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
     $column['escaped'] = $column['escaped'] ?? false;
     $column['prefix'] = $column['prefix'] ?? '';
     $column['suffix'] = $column['suffix'] ?? ' items';
-    $column['text'] = '';
+    $column['text'] = $column['default'] ?? '-';
+
+    if(is_callable($column['value'])) {
+        $column['value'] = $column['value']($entry);
+    }
 
-    if($value) {
-        $column['text'] = $column['prefix'].$value.$column['suffix'];
+    if(!empty($column['value'])) {
+        $column['text'] = $column['prefix'].count($column['value']).$column['suffix'];
```

Comment:
Ok scratch that, I changed my mind. It's more consistent to show `0 items` - more important to be consistent _with itself_ than with _other columns_. I'm ok with how this is now, showing `0 items`.

Reply:
No! 😅 I think you are right ...
`array_count` works the same way, I think we should keep the cell empty ...

Reply:
But this raises an issue
![image](https://user-images.githubusercontent.com/1838187/142767445-62154a23-84e0-4686-92a8-d185fda57439.png)

If we have a wrapper with a link, the link will be there with the dash, should it?

Reply:
And this may happen in other fields where we just added the dash `-` 😬
Maybe I should refactor this a little bit more so that;

```php
@if(!empty($column['value']))

    @includeWhen(!empty($column['wrapper']), 'crud::columns.inc.wrapper_start')
        {{ $column['prefix'] }}

        @if($column['escaped'])
            {{ $column['value'] }}
        @else
            {!! $column['value'] !!}
        @endif

        {{ $column['suffix'] }}
    @includeWhen(!empty($column['wrapper']), 'crud::columns.inc.wrapper_end')

@else
    {{ $column['default'] ?? '-' }}
@endif
```

This way, the empty dash will always be outside the wrapper ...

Let me know @tabacitu ✌, I know in advance – we shouldn't be worrying that much with perfection – but let's hope this is the last change ✌😅